base/poco/Crypto/include/Poco/Crypto/EVPPKey.h

```diff
@@ -188,8 +188,9 @@ namespace Crypto
                     pFile = fopen(keyFile.c_str(), "r");
                     if (pFile)
                     {
-                        pem_password_cb * pCB = pass.empty() ? (pem_password_cb *)0 : &passCB;
-                        void * pPassword = pass.empty() ? (void *)0 : (void *)pass.c_str();
+                        pem_password_cb * pCB = &passCB;
+                        static constexpr char * no_password = "";
```

Comment:
I think this change will pCB be called always. What is the reason for that?

Reply:
It's really strange (and annoying how openSSL handles these cases)
If password is nullptr, then it fallbacks to a default handler which gives you a prompt to enter passphrase and waits for input from stdin and that really doesn't work with a background process like Keeper
I want to have an empty password valid entry, i.e. if a private key is protected by passphrase and password is empty, it should be decrypted (and fail) with empty passphrase
so I changed to code to always call `passCB` even if the password is empty.